ludwig/models/modules/embedding_modules.py

```diff
@@ -163,6 +163,8 @@ def __init__(
     ):
         super(EmbedWeighted, self).__init__()
 
+        self.vocab = vocab
```

Comment:
not super sure why you need this, is self.vocab used somewhere else?

Reply:
It's referenced in line 203.

Reply:
You are right. But there we are only using the length, so maybe it's worth just doing:
```
self.vocab_length = len(vocab)
```
and then using it in line 203.

Reply:
Makes sense, updated.